Upstream patch to change the client binary's name to clFFT-client.
Remove for clfft-9999 as soon as develop branch is merged into master.


From f3cbd72953094217dfeb41c4e2a328d4b4a494e0 Mon Sep 17 00:00:00 2001
From: Marius Brehler <marius.brehler@tu-dortmund.de>
Date: Wed, 18 Feb 2015 09:15:13 +0100
Subject: [PATCH] Rename Client to clFFT-client

---
 src/client/CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/client/CMakeLists.txt b/src/client/CMakeLists.txt
index fdf209f..3b38b95 100644
--- a/src/client/CMakeLists.txt
+++ b/src/client/CMakeLists.txt
@@ -43,15 +43,15 @@ endif( )
 # Include standard OpenCL headers
 include_directories( ${Boost_INCLUDE_DIRS} ${OPENCL_INCLUDE_DIRS} ../../../common ${PROJECT_BINARY_DIR}/include ../include )
 
-add_executable( Client ${Client.Files} )
+add_executable( clFFT-client ${Client.Files} )
 
-target_link_libraries( Client clFFT ${Boost_LIBRARIES} ${OPENCL_LIBRARIES} ${DL_LIB} )
+target_link_libraries( clFFT-client clFFT ${Boost_LIBRARIES} ${OPENCL_LIBRARIES} ${DL_LIB} )
 
-set_target_properties( Client PROPERTIES VERSION ${CLFFT_VERSION} )
-set_target_properties( Client PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/staging" )
+set_target_properties( clFFT-client PROPERTIES VERSION ${CLFFT_VERSION} )
+set_target_properties( clFFT-client PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/staging" )
 
 # CPack configuration; include the executable into the package
-install( TARGETS Client
+install( TARGETS clFFT-client
         RUNTIME DESTINATION bin${SUFFIX_BIN}
         LIBRARY DESTINATION lib${SUFFIX_LIB}
         ARCHIVE DESTINATION lib${SUFFIX_LIB}/import
-- 
2.0.4

